controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java

```diff
@@ -142,14 +143,6 @@ protected final void shutDown() throws Exception {
                 } catch (Exception e) {
                     log.info("Exception while closing EventProcessorCell reader from checkpointStore: {}.", e.getMessage());
                 }
-
-                // Next, clean up the reader and its position from checkpoint store
-                log.info("Cleaning up checkpoint store for {}", objectId);
-                try {
-                    checkpointStore.removeReader(process, readerGroupName, readerId);
```

Comment:
why are we removing this.. this is a zookeeper node which needs to be cleaned up.

Reply:
I moved it into checkpointStore:
https://github.com/pravega/pravega/pull/4999/files#diff-36211c36a9b83866c7fae956d50dce693373d12d9d9e29bf550bd4d52b52851bR161-R164